config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -786,6 +826,37 @@ periodics:
   annotations:
     testgrid-dashboards: sig-node-containerd
     testgrid-tab-name: node-e2e-features
+- name: ci-cri-containerd-node-e2e-unlabelled
```

Comment:
why renaming from orphans?

Reply:
Because it's a pretty insensitive name; new name is more descriptive and inclusive.